tools/Dockerfile

```diff
@@ -81,25 +58,36 @@ RUN cp -p LICENSE /usr/share/licenses/eksctl && \
       --out-dir /usr/share/licenses/eksctl/vendor \
       go-vendor ./vendor
 RUN curl -L "${EKSCTL_BINARY_URL}" \
-      -o eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz && \
-    grep eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz \
+      -o "eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz" && \
+    grep "eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz" \
       /hashes/eksctl | sha512sum --check - && \
-    tar -xf eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz -C /tmp && \
-    rm eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz
+    tar -xf "eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz" -C /tmp && \
+    rm "eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz"
+
+# =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
+FROM build-go as eksctl-anywhere-build
+
+USER root
+RUN mkdir -p /usr/share/licenses/eksctl-anywhere && \
+    chown -R builder:builder /usr/share/licenses/eksctl-anywhere
+
+ARG EKSA_VERSION=0.11.4
+ARG EKSA_SOURCE_URL="https://github.com/aws/eks-anywhere/archive/refs/tags/v${EKSA_VERSION}.tar.gz"
+
+ARG EKSA_RELEASE_NUMBER=21
+# TODO EKS-A currently does not provide an ARM64 binary, add the hash for it when they do
+ARG EKSA_BINARY_URL="https://anywhere-assets.eks.amazonaws.com/releases/eks-a/${EKSA_RELEASE_NUMBER}/artifacts/eks-a/v${EKSA_VERSION}/${GOOS}/amd64/eksctl-anywhere-v${EKSA_VERSION}-${GOOS}-amd64.tar.gz"
 
-# eksctl-anywhere
 # TODO Try and build this; it's not straightforward with the tight coupling between EKS-A releases and its bundles
 WORKDIR /home/builder/
-RUN mkdir eksctl-anywhere && curl -L ${EKSA_SOURCE_URL} \
-      -o eksctl_anywhere_${EKSA_VERSION}.tar.gz && \
-    grep eksctl_anywhere_${EKSA_VERSION}.tar.gz \
+RUN mkdir eksctl-anywhere && curl -L "${EKSA_SOURCE_URL}" \
+      -o "eksctl_anywhere_${EKSA_VERSION}.tar.gz" && \
+    grep "eksctl_anywhere_${EKSA_VERSION}.tar.gz" \
       /hashes/eksctl | sha512sum --check - && \
-    tar -xf eksctl_anywhere_${EKSA_VERSION}.tar.gz --strip-components 1 -C eksctl-anywhere && \
-    rm eksctl_anywhere_${EKSA_VERSION}.tar.gz && \
-    sed -i 's/v0.2.0-alpha.6/v0.2.1/' eksctl-anywhere/go.mod && \
-    rm eksctl-anywhere/go.sum
+    tar -xf "eksctl_anywhere_${EKSA_VERSION}.tar.gz" --strip-components 1 -C eksctl-anywhere && \
+    rm "eksctl_anywhere_${EKSA_VERSION}.tar.gz"
 
-USER root
+USER builder
```

Comment:
I'm pretty sure this has to be root or we'll run into permission issues.

Reply:
`/usr/share/licenses/eksctl-anywhere` is `chown`ed to **builder** so that shouldn't be the case. That said, it looks like this line needs to move to L81 which would explain any permissions issues you might have seen in the past (see L119). I'll post a revision!

Reply:
The issue was extracting the binary to `/tmp`:

```
#37 1.217 eksctl_anywhere_0.12.1_23_linux_amd64.tar.gz: OK
#37 1.686 tar: .: Cannot utime: Operation not permitted
#37 1.686 tar: .: Cannot change mode to rwxr-xr-t: Operation not permitted
```